docs/userguide/main.md

```diff
@@ -327,14 +327,7 @@ Or custom ones, for example that Aiven plugin for prometheus-exporter:
 ```
 
 ## Custom init helper
-The initHelper is an image used during cluster setup. By default OpensearchOperator creates an init container with that image (_public.ecr.aws/opsterio/busybox:1.27.2-buildx_), according to change that behaviour and use a custom image, follow the instructions. Globally used image repository is set using the `DefaultRepo` option:
-
-```yaml
-  spec:
-    general:
-      defaultRepo: "mycustomrepo.cr"
-```
-
+The initHelper is an image used during cluster setup. By default the OpensearchOperator creates an init container with that image (`public.ecr.aws/opsterio/busybox:1.27.2-buildx`, a normal busybox image). To change that behaviour and use a custom image for the initHelper, add the following to your cluster spec:
 Alternatively, a custom image for just the initHelper (busybox used during cluster setup) is configured as follows:
```

Comment:
This line is redundant

Reply:
which of them>?

Reply:
The line `Alternatively, a custom image for just the initHelper (busybox used during cluster setup) is configured as follows:`
All the information is already in the line above.